.ocamlformat

```diff
@@ -0,0 +1,6 @@
+profile = default
+version = 0.22.4
+
```

Comment:
See: https://github.com/rescript-lang/rescript-vscode/blob/master/analysis/.ocamlformat
There's nothing set in stone in that file, just that's been used so far.
Seems like a good idea to converge to one, and use it in all places.

Reply:
I have taken over the additional settings from there now. I am trying to minimize the changes compared to the existing code style.

I am wondering if it is possible to get rid of the space before the `:` in

```ocaml
val loc : t -> Location.t
```

Haven't found the setting for that yet (only for `let` bindings).